packages/fop.rb

```diff
@@ -15,21 +15,19 @@ def self.install
     system "mkdir -p #{CREW_DEST_PREFIX}/bin"
     system "mkdir -p #{CREW_DEST_LIB_PREFIX}/fop"
     system "cp -r . #{CREW_DEST_LIB_PREFIX}/fop"
-    system "chmod +x #{CREW_DEST_LIB_PREFIX}/fop/fop/fop"
     FileUtils.cd("#{CREW_DEST_PREFIX}/bin") do
       system "echo '#!/bin/bash' > fop"
-      system "echo 'PWD=$(pwd)' >> fop"
       system "echo 'cd #{CREW_LIB_PREFIX}/fop/fop' >> fop"
       system "echo './fop \"$@\"' >> fop"
       system "echo 'cd $PWD' >> fop"
       system "chmod +x fop"
     end
-  end
 
-  def self.postinstall
-    puts
-    puts "To complete the installation, set the environment variable as follows:".lightblue
-    puts "echo 'export JAVA_HOME=#{CREW_LIB_PREFIX}/jdk8' >> ~/.bashrc && source ~/.bashrc".lightblue
-    puts
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
+    @env = <<~EOF
+      # Fop configuration
+      export JAVA_HOME=#{CREW_LIB_PREFIX}/jdk8
+    EOF
+    IO.write("#{CREW_DEST_PREFIX}/etc/env.d/fop", @env)
```

Comment:
Bump fop version?

Reply:
Remember, we're storing in the `CREW_DEST_PREFIX` directory so that means a new file is tracked in the package metadata.